### PR TITLE
Added some discord channels as well one newsletter + other

### PR DESCRIPTION
**Pull request by fklement**
Started with some addings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,16 @@
   - [Developer Hangout](https://discord.gg/developers)
   - [The Coding Den](https://discord.gg/code)
   - [The Programmer's Hangout](https://discord.gg/BQN6BYE)
+  - [Functional Programming](https://discord.gg/E356PM)
+  - [Python](https://discord.gg/python)
+  - [Programmer's Hangout](https://discord.gg/programming)
```
Comment (b3u) [inline README.md]: this is a duplicate entry

Comment (fklement) [reply README.md]: Oh damn, you're right.  
But I would prefer to go with the /programming one, as the other is a personalised invitation?

**Comment by b3u** [comment]
@fklement Thanks for the pr. I really appreciate it. Looking at it now.

File: README.md (hunk 1 above)
Comment (b3u) [reply README.md]: true

**Comment by fklement** [comment]
> good ideas. please remove one of the TPH links.

Yes I'll do that by the end of today.

**Comment by b3u** [comment]
@fklement i'll do it. Thanks for the PR!

File: README.md (hunk 1 above)
Comment (b3u) [reply README.md]: i'll fix it

**Pull request merged**